server/src/main/java/org/spine3/server/stand/Stand.java

```diff
@@ -0,0 +1,663 @@
+            typeToRepositoryMap.put(entityType, (EntityRepository<I, E, ? extends Message>) repository);
+        }
+        if (repository instanceof AggregateRepository) {
+            knownAggregateTypes.add(entityType);
+        }
+
+    }
+
+    /**
+     * Dumps all {@link TypeUrl}-to-{@link EntityRepository} relations.
+     */
+    @Override
+    public void close() throws Exception {
+        typeToRepositoryMap.clear();
+        storage.close();
+    }
+
+    /**
+     * A contract for the callbacks to be executed upon entity state change.
+     *
+     * @see #activate(Subscription, StandUpdateCallback)
+     * @see #cancel(Subscription)
+     */
+    public interface StandUpdateCallback {
+
+        void onEntityStateUpdate(Any newEntityState);
+    }
+
+
+    public static class Builder {
+        private StandStorage storage;
+        private Executor callbackExecutor;
+
+
+        /**
+         * Set an instance of {@link StandStorage} to be used to persist the latest an Aggregate states.
+         *
+         * <p>If no {@code storage} is assigned, the {@link InMemoryStandStorage} is be set by default.
+         *
+         * @param storage an instance of {@code StandStorage}
+         * @return this instance of {@code Builder}
+         */
+        public Builder setStorage(StandStorage storage) {
+            this.storage = storage;
+            return this;
+        }
+
+
+        public Executor getCallbackExecutor() {
+            return callbackExecutor;
+        }
+
+        /**
+         * Set an {@code Executor} to be used for executing callback methods.
+         *
+         * <p>If the {@code Executor} is not set, {@link MoreExecutors#directExecutor()} will be used.
+         *
+         * @param callbackExecutor the instance of {@code Executor}
+         * @return this instance of {@code Builder}
+         */
+        public Builder setCallbackExecutor(Executor callbackExecutor) {
+            this.callbackExecutor = callbackExecutor;
+            return this;
+        }
+
+        public StandStorage getStorage() {
+            return storage;
+        }
+
+
+        /**
+         * Build an instance of {@code Stand}.
+         *
+         * @return the instance of Stand
+         */
+        public Stand build() {
+
+            if (storage == null) {
+                storage = InMemoryStandStorage.newBuilder()
+                                              .build();
+            }
+
+            if (callbackExecutor == null) {
+                callbackExecutor = MoreExecutors.directExecutor();
+            }
+
+            final Stand result = new Stand(this);
+            return result;
+        }
+    }
+
+    /**
+     * Registry for subscription management.
+     *
+     * <p>Provides a quick access to the subscription records by {@link TypeUrl}.
+     * <p>Responsible for {@link Subscription} object instantiation.
+     */
+    private static final class StandSubscriptionRegistry {
```

Comment:
It can be a package level class called just `SubscriptionRegistry`. Also, please consider having `SubscriptionRecord` at the package level. This way it would be easier to test.